cmd/daemon/main.go

```diff
@@ -64,14 +64,15 @@ import (
 
 // Values set when building the application
 var (
-	Salt        = ""
-	Version     = "0.0.0"
-	Environment = ""
-	PackageType = ""
-	Arch        = ""
-	Port        = 6960
-	ConnType    = "unix"
-	ConnURL     = internal.DaemonSocket
+	Salt          = ""
+	Version       = "0.0.0"
+	Environment   = ""
+	PackageType   = ""
+	Arch          = ""
+	Port          = 6960
+	ConnType      = "unix"
+	ConnURL       = internal.DaemonSocket
+	FirebaseToken = "" // If this is moved to another package the scripts need to be updated
```

Comment:
Can this be moved to `cmd/daemon/vpnconfig_telio.go`?

Reply:
yes it can be, was only thinking if it would be easier to keep with the other variables that are set at build time
`// Values set when building the application`